pkg/cluster/inventory.go

```diff
@@ -187,6 +188,44 @@ func (i *DefaultInventory) createStatus(configEntity *model.ClusterConfiguration
 		return nil, err
 	}
 
+	// create new reconcile pending statuses
+	if status == model.ClusterStatusReconcilePending {
+		q, err := db.NewQuery(i.Conn, &model.ClusterStatusEntity{}, i.Logger)
+		if err != nil {
+			return nil, err
+		}
+
+		notReconciledStatues, err := q.Select().
```

Comment:
We should not always query the whole history.  Would propose following flow:

* Implement DB migration logic (will require #442 ) and migrate all ol pending-status which were not applied to `skipped`
* Check just the previous cluster-state status, and if this is `pending`, just change this one to `skipped`